docs/brotli.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><g transform="translate(256,256)scale(24)rotate(15)" style="fill:#fff;"><circle style="fill:#ea3" r="10"/><ellipse rx="2" ry="8"/><ellipse cx="5" rx="1" ry="4"/><ellipse cx="-5" rx="1" ry="4"/></g></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><g transform="translate(256,256)scale(24)rotate(15)" style="fill:#fff"><circle style="fill:#ea3" r="10"/><ellipse rx="2" ry="8"/><ellipse cx="5" rx="1" ry="4"/><ellipse cx="-5" rx="1" ry="4"/></g></svg>
```

Comment:
@eustas This can be reduced even more:

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="#fff" viewBox="0 0 512 512"><g transform="rotate(15 -844 1100)scale(24)"><circle fill="#ea3" r="10"/><ellipse rx="2" ry="8"/><ellipse rx="1" ry="4" cx="5"/><ellipse rx="1" ry="4" cx="-5"/></g></svg>
```

Reply:
Cool! Will create a PR soon. Thanks!